lib/NormalModule.js

```diff
@@ -231,7 +231,7 @@ class NormalModule extends Module {
 			}
 
 			if(err) {
-				const error = new ModuleBuildError(this, err, this.getCurrentLoaderName(loaderContext));
+				const error = new ModuleBuildError(this, err, this.purifyCurrentLoaderName(loaderContext));
```

Comment:
Maybe better to make the third argument an `{Object}`, so it can be extended in the future (not directly related to this PR though)

```js
new ModuleBuildError(this, err, { loader: this.getLoader(), ... })
```

Reply:
That's true

Reply:
No real preference for the `key` name here :)